core/src/main/java/io/grpc/transport/ServerTransportListener.java

```diff
@@ -48,4 +49,9 @@
    */
   ServerStreamListener streamCreated(ServerStream stream, String method,
       Metadata.Headers headers);
+
+  /**
+   * The transport completed shutting down. All resources have been released.
+   */
+  void transportTerminated();
```

Comment:
I am using ```shutdown``` to indicate the user has shutdown the service and ```terminated``` indicating it has wound down. The problem with "stopping" is that it is a temporal state; when you have stopped you are no longer stopping. I want a word that is permanent, even after termination. And shutdown is an appropriate word for such usage in English.